app/forms/idv/finance_form.rb

```diff
@@ -1,34 +1,39 @@
 module Idv
   class FinanceForm
     include ActiveModel::Model
+    include FormFinanceValidator
 
     FINANCE_TYPES = [:ccn, :mortgage, :home_equity_line, :auto_loan].freeze
 
     FINANCE_HTML_OPTIONS = {
-      ccn: { maxlength: 8 }
+      ccn: {
+        minlength: FormFinanceValidator::VALID_MINIMUM_LENGTH,
+        maxlength: FormFinanceValidator::VALID_MINIMUM_LENGTH
```

Comment:
since ccn is different than the rest, I might consider a special `CCN_LENGTH` constant? otherwise it might look like a copy-paste error?

But not a big del